### PR TITLE
Refactor runner and commands.

### PR DESCRIPTION
**Pull request by matanlurey**
No breaking changes to the package.

### CODE REVIEW

File: mono_repo/lib/mono_repo.dart [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
+import 'dart:async';
+
 import 'package:args/command_runner.dart';
 
-import 'src/check_command.dart';
-import 'src/init_command.dart';
-import 'src/presubmit_command.dart';
-import 'src/pub_command.dart';
-import 'src/travis_command.dart';
+import 'package:mono_repo/src/commands/check.dart';
```
Comment (kevmoo) [inline mono_repo/lib/mono_repo.dart]: keep these are relative urls, please

File: mono_repo/lib/src/runner.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import 'package:args/command_runner.dart';
+
+import 'commands/check.dart';
+import 'commands/init.dart';
+import 'commands/presubmit.dart';
+import 'commands/pub.dart';
+import 'commands/travis.dart';
+
+const _name = 'mono_repo';
```
Comment (kevmoo) [inline mono_repo/lib/src/runner.dart]: What's the upside to creating these consts – vs just plugging the literals in the `super` call?

File: mono_repo/lib/mono_repo.dart (hunk 1 above)
Comment (matanlurey) [reply mono_repo/lib/mono_repo.dart]: Done.

File: mono_repo/lib/src/runner.dart (hunk 2 above)
Comment (matanlurey) [reply mono_repo/lib/src/runner.dart]: Not significant. Done.

**Comment by kevmoo** [comment]
Squash and merge away! (when travis is happy)

**Pull request merged**